Classes/Units/Controllers/Inventory.js

```diff
@@ -109,7 +138,7 @@ class Inventory {
 			return false;
 
 		var Item = ItemList[this.Items[slot].id];
-		this.parent.stats.Gold += Item.GoldCost * 0.4;
+		this.parent.stats.Gold += Item.GoldSell;
```

Comment:
Usually item gold sell amount is 70% but not for all items.
https://leagueoflegends.fandom.com/wiki/Shop#Item_Sale

So let's make it this way, there will be no need for setting it for every item:
`this.parent.stats.Gold += Item.GoldSell ?? (Item.GoldCost * 0.7);`